src/ansys/pytwin/evaluate/twin_model.py

```diff
@@ -1804,6 +1853,87 @@ def generate_points(self, rom_name: str, on_disk: bool = True, named_selection:
             msg += f"\n{str(e)}."
             self._raise_error(msg)
 
+    def project_tbrom_on_mesh(self, rom_name: str, mesh: pv.DataSet, interpolate: bool, named_selection: str = None):
+        """
+        Project the field ROM data onto a mesh
```

Comment:
Details docstring (see comment for `tbrom.project_on_mesh` method)

Reply:
added more info